src/DistributedMetadata/DDLService.cpp

```diff
@@ -331,7 +374,7 @@ void DDLService::createTable(IDistributedWriteAheadLog::RecordPtr record)
         assert(!hosts.empty());
 
         std::vector<Poco::URI> target_hosts{
-            toURIs(hosts, fmt::format(DDL_TABLE_POST_API_PATH_FMT, database, getURIEndpoint(record->headers)), http_port)};
+            toURIs(hosts, fmt::format(DDL_TABLE_POST_API_PATH_FMT, database, getTableCategory(record->headers)), http_port)};
```

Comment:
we can refactor the `fmt::format` to use `getTableApiPath` ?

Reply:
Yes